Yoast/wordpress-seo
# Don't canonicalize when we noindex

## Summary

This PR can be summarized in the following changelog entry:

* When we set `noindex` on a page, we no longer add a canonical, to prevent confusing search engines.

## Relevant technical choices:

* Couldn't properly unit test this, because, well... `class-frontend.php`.

## Test instructions

This PR can be tested by following these steps:

* Set a post to `noindex`, you should see the canonical in the source disappear.

## Quality assurance


Fixes #8719
